docker-compose.yaml

```diff
@@ -1,10 +1,22 @@
 version: '3.9'
+
+x-defaults:
+  - yugabyte_image: &yb-image ${YB_IMAGE:-yugabytedb/yugabyte:2.15.3.0-b231}
```

Comment:
I had to build my own yugabyte docker image for linux/arm64, so I made it an env var to allow overrides.